components/bl-autocomplete-component/src/helpers/validate.js

```diff
@@ -5,13 +5,20 @@ export const validate = items => {
   
   if (typeof items === 'string') {
     return items.split(',').map(item => ({
-      objectId: item,
+      objectId: BackendlessUI.UUID.short(),
       value: item,
       label: item,
     }));
   }
     
-  if (typeof items === 'object') {
-    return Object.values(items);
+  if (Array.isArray(items)) {
```

Comment:
If the first item hasn't objectId, that does not guarantee all others haven't.
In any case, as I said earlier, we must avoid work with objectId. Options could comes not from Backednless